R/shiny-options.R

```diff
@@ -246,10 +246,16 @@ shinyOptions <- function(...) {
 captureAppOptions <- function() {
   options <- list(
     appDir = getwd(),
-    bookmarkStore = getShinyOption("bookmarkStore")
+    bookmarkStore = getShinyOption("bookmarkStore"),
+    bootstrapTheme = getShinyOption("bootstrapTheme"),
```

Comment:
Is it necessary to capture these options?

Reply:
I was thinking it might be for `themeDependencyFuncs` to avoid a situation where creating two apps then running one of them would call all the registered functions (for both apps), but I suppose that's not a problem as long as we [do the registration in `tagFunction()` and check to make sure `shiny::isRunning()`](https://github.com/rstudio/bootstraplib/pull/120/files#diff-68e15f5a86c76d37b4d079274c90aa4fR137)